scene/resources/tile_set.cpp

```diff
@@ -212,7 +212,7 @@ bool TileSet::_get(const StringName &p_name, Variant &r_ret) const {
 			Vector3 v;
 			for (Map<Vector2, int>::Element *E = tile_map[id].autotile_data.priority_map.front(); E; E = E->next()) {
 				if (E->value() > 1) {
-					//Don't save default value
+					//Dont save default value
```

Comment:
Heh, you added a typo here, the previous one was correct.

Reply:
Ah, maybe it was fixed in `master` after you started working on it, and you undid the fix when rebasing your work for this new PR.

Reply:
At any rate, it's not a big issue, you can leave it like this. It will be fixed eventually by `codespell`.

Reply:
It's too late, I've pushed the fixed one.